vumi/persist/tests/test_txriak_manager.py

```diff
@@ -217,19 +206,22 @@ def test_run_riak_map_reduce_with_timeout(self):
             function(value, keyData) {
                 var date = new Date();
                 var curDate = null;
+                // Wait 11ms so we run past the 10ms timeout.
                 do { curDate = new Date(); }
                 while(curDate-date < 11);
+                return value;
             }
             """)
 
         try:
             yield self.manager.run_map_reduce(mr, lambda m, l: None)
         except Exception, err:
-            msg = unrepr_string(str(err))
-            self.assertTrue(msg.startswith(
-                "Error running MapReduce operation."))
-            self.assertTrue(msg.endswith(
-                "Body: '{\"error\":\"timeout\"}'"))
+            msg = str(err)[1:-1].decode("string-escape")
+            if not all([
```

Comment:
I actually quite like the `all()` style because you can think of it as a list of conditions to be satisfied. Otoh, `and` is simpler. Don't want to get stuck on this tiny detail - I was mostly just curious if this was a common Python thing people did...